.gitignore

```diff
@@ -31,23 +31,9 @@ build/
 ### VS Code ###
 .vscode/
 
-src/main/resources/application.properties
-
-src/main/resources/config.json
-
 *.db
 
-src/main/resources/application.fieldlab.properties
-
-src/main/resources/application.local.properties
-
-src/main/resources/application.swf.properties
-
-src/main/resources/config.local.json
-
-src/main/resources/config.swf.json
-
-TODO.txt
+src/main/resources/*.properties
+src/main/resources/*.json
 
-bouw.bat
-/schema/Zaak_DocumentServices_1_1_02
+/src/test/resources/ladybug/**
```

Comment:
Ladybug resource should only change when you replace a report in the GUI. Let's investigate in a separate issue

Reply:
great!